tests/src/python/test_qgsvectorlayereditutils.py

```diff
@@ -197,7 +197,7 @@ def testAddRingV2(self):
         vle = QgsVectorLayerEditUtils(layer)
         result = vle.addRingV2(QgsLineString([QgsPoint(3, 3), QgsPoint(3, 4), QgsPoint(4, 4), QgsPoint(4, 3), QgsPoint(3, 3)]))
         assert Qgis.GeometryOperationResult.Success == result[0]
-        assert [2, 1] == result[1]
+        assert [1, 2] == sorted(result[1])
```

Comment:
self.assertEqual(set(2, 1), set(result[1]))

Reply:
I've included this change in my PR https://github.com/qgis/QGIS/pull/50592 - if we get there earlier

Reply:
Sorry the correct syntax is:
> self.assertEqual({2, 1}, set(result[1]))